packages/firestore/test/unit/util/async_queue.test.ts

```diff
@@ -90,10 +90,38 @@ describe('AsyncQueue', () => {
       )
       .then(() => {
         expect(queue.failure).to.equal(expected);
-      })
-      .then(() => {
-        setLogLevel(oldLogLevel);
       });
+
+    // Schedule a second failing operation (before the first one has actually
+    // executed and failed). It should not be run.
+    const op2Promise = queue
+      .schedule(() => {
+        return defer(() => {
+          expect.fail('op2 should not be executed.');
+        });
+      })
+      .then(
+        () => {
+          expect.fail('expected an error');
```

Comment:
Maybe localize these failures to which operation they apply to? e.g. 'op2 expected an error' here and 'op1 ...' above?

Reply:
Done.